notebooks/en/rag_with_hugging_face_gemma_elasticsearch.ipynb

```diff
@@ -0,0 +1,6328 @@
+{
```

Comment:
<p>Blank cell should be deleted here</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/huggingface/cookbook/pull/124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>👍</p>